Mathlib/MeasureTheory/Decomposition/RadonNikodym.lean

```diff
@@ -407,6 +407,66 @@ lemma rnDeriv_le_one_of_le (hμν : μ ≤ ν) [SigmaFinite ν] : μ.rnDeriv ν
   simp only [Pi.one_apply, MeasureTheory.set_lintegral_one]
   exact (Measure.set_lintegral_rnDeriv_le s).trans (hμν s)
 
+section MeasurableEmbedding
+
+variable {mβ : MeasurableSpace β} {f : α → β}
+
+/-! The results in this section are stated for finite measures, because we need the pushforwards of
+some measures to have a Lebesgue decomposition with respect to the pushforward of other measures
+(for example `(μ.singularPart ν).map f` w.r.t. `ν.map f`). It would be sufficient to know that all
+those pushforwards are σ-finite, but `μ.map f` may not be σ-finite even if `μ` is. However, if `μ`
```

Comment:
I'm not sure I get this comment: in this section, `f` is always a measurable embedding, so it should map sigma-finite measures to sigma-finite measures, right?